src/librustc/session/config.rs

```diff
@@ -709,6 +709,36 @@ pub fn append_configuration(cfg: &mut ast::CrateConfig,
     }
 }
 
+unsafe fn configure_llvm(sess: &Session) {
```

Comment:
Let's keep this where it lived initially and perhaps just call it from the driver instead